source/reference/minio-mc/mc-support-inspect.rst

```diff
@@ -0,0 +1,143 @@
+======================
+``mc support inspect``
+======================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+.. mc:: mc support inspect
+
+Description
+-----------
+
+:mc-cmd:`mc support inspect` allows you to collect files, including metadata files, from all backend drives based on a specified path. 
+The command produces a zip file that includes all matching files with their respective *host+drive+path*. 
+
+MinIO Engineering may request this output as part of diagnostics in |subnet|.
+
+Use the :mc-cmd:`mc support inspect` command to obtain raw object contents for analysis and upload it to the MinIO Subscription Network, |subnet|.
+Or use the command to download metadata for objects.
+
+You can read the resulting object using the MinIO debugging tool, described below.
+Independent or third-party use of the output for diagnostics or remediation is done at your own risk.
+
+.. note::
+   
+   :mc-cmd:`mc support inspect` requires a MinIO host from October 2021 or later for full functionality. 
+
+Install additional tools to decrypt or decode downloaded files.
+Instructions are included below.
+
+
+Wildcards
+---------
+
+The command allows for the wildcard character, `\`*\``. 
+
+.. code-block:: shell
+   :class: copyable
+
+   mc support inspect ALIAS/bucket/path/**/xl.meta
+   
+This command collects all ``xl.meta`` recursively.
+
+
+Examples
+--------
+
+Download Metadata for an Object from All Drives
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can download the metadata for an object.
+Metadata stores in an ``xl.meta`` binary file.
+
+- MinIO alias: ``minio1``
+- Bucket: ``mybucket``
+- Object: ``myobject``
+
+The file downloads from all drives as a zip archive file.
+
+.. code-block:: shell
+   :class: copyable
+
+   mc support inspect minio1/mybucket/myobject/xl.meta
+
+You can convert the contents of an ``xl.meta`` file to JSON format with the :ref:`decode tool <minio-xl-meta-decode-tool>`.
+
+Download All Parts of an Object as an Encrypted Zip
```

Comment:
We're missing a bit of parity here - sometimes ew talk about downloading the metadata, others all parts - but the command is identical other than encryption.

I think xl.meta is *just* the metadata, not the object itself. If so we should state that as the only thing downloaded and make that consistent on this page.